lint_test/avoid_late_keyword_allow_initialized_test/pubspec.yaml

```diff
@@ -0,0 +1,13 @@
+name: avoid_late_keyword_allow_initialized_test
+publish_to: none
+
+environment:
+  sdk: '>=3.0.0 <4.0.0'
+
+dependencies:
+  flutter:
+    sdk: flutter
```

Comment:
```suggestion
```

Reply:
Yes, I did it yesterday but create PR only today